src/browser/HostInstaller.h

```diff
@@ -35,9 +35,12 @@ class HostInstaller : public QObject
         FIREFOX = 2,
         VIVALDI = 3,
         TOR_BROWSER = 4,
-        BRAVE = 5
+        BRAVE = 5,
+        EDGE = 6
     };
 
+    static const int SUPPORTED_BROWSERS_COUNT = 7;
```

Comment:
I know of several pieces of software where this number is made the last value of the enum but that's preference I guess (and depending on the code style of this project which I am not familiar with)

Reply:
I agree with this approach.